examples/cpp_examples/hello-embed.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+This file is an example embedding for the Interpreter Manager class
+*/
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "interpreterManager.h"
+
+
+extern "C" {
+    #include "py/compile.h"
+    #include "py/runtime.h"
+    #include "py/gc.h"
+    #include "py/stackctrl.h"
+}
```

Comment:
Do we need to include `gc.h` & `stackctrl.h"? Can we remove them if not used? Also these headers will be included from `interpreterManager.h` header.